### ISSUE TITLE
Numerical instability at 290

### ISSUE THREAD

**Issue by moondrop-bagram** [firemodels/fds#7304]
Hello,

The following lines of code creates a numerical instability at 290s. At one of the windows, a large increase of velocity is observed. The materials near the window have not yet burnt away so I'm unable to figure out why this happens. 

I originally had the whole floor as a fire VENT and the OBST having an IGNITION_TEMPERATURE=400. This resulted in a numerical instability at around 1000s. I also tried closing all the windows for the code below and having just a door VENT and there's a numerical instability at 230s.

[geotest.fds.txt](https://github.com/firemodels/fds/files/3102960/geotest.fds.txt)

**Comment by mcgratta** [comment]
The case is too complicated to diagnose easily. Are there pockets of enclosed volumes somewhere in the domain? Can you run the case with only one object burning away?

**Comment by moondrop-bagram** [comment]
Hello,

I just realized the problem a few hours ago. Indeed the tables (which look like blocks) have pockets of air enclosed in them. I sorted it out by removing the top of the table and did not experience any increased velocities at those locations. However, now there's a large velocity increase in the wooden cabinet (the one at X=5.5, Y=0). I believe this is because the cabinet is not exactly at y=0 but at y=0.05 which again, is causing some air flow issues when some of the material is burning away. Are there be any provisions in FDS to proceed with the calculations since I know they are essentially not violating the behavior at that location? (For example, the pocket of air may cause the velocity increase but that is also the true behavior in real life. Would there be any means of differentiating unintended numerical instabilities and intended increased flows due to enclosed volumes. If FDS doesn't allow this right now, IMHO wouldn't it be a valuable addition in the future.) 

I will simplify the entire model by merely having blocks of wood of the same volume but whose density is altered to give the same weight as the present model. However would that simplification accurately represent the burning behavior of the current model? And in the near future, I'll perform a sensitivity analysis as a research paper for the numerical instabilities caused depending on the mesh size and VENT opening area (for an OBST bounding pockets of air -- representing hollow wooden boxes). 

Thank you so much for looking into it @mcgratta  :)
